Final project cleanup, adding rehearsal simulation example and documentation

aschellenberg74 opened the pull request.

aschellenberg74 committed:
final project cleanup and adding rehearsal simulation example

aschellenberg74 committed:
adding documentation (general and rehearsal simulation)

michaelcdevin reviewed (commented):
Thanks @aschellenberg74 for getting this all wrapped up and organized! Apologies for taking so long to get to this, it's been a busy last few weeks as you can imagine :)

Overall this looks really good, just a few small requests before we merge it in:
* Can you remove the `OpenFAST-Simulink_x64.dll`, `OpenFAST-Simulink_x64.lib`, and `FAST_SFunc.mexw64` files? Just want to avoid having libraries for another program that we don't own contained in the repo
* Can you remove the `Example/OpenFAST models/IEA15MW/Airfoils` folder as well? There's a lot of files here that we don't need since we aren't simulating wind in the rehearsal simulation (i.e. it shouldn't have any effect on the rehearsal simulation)
* One code comment about `moordyn_sfunc_3dof.cpp`, see below

michaelcdevin commented on MATLAB/moordyn_sfunc_3dof.cpp:
```diff
@@ -195,79 +196,78 @@ static void mdlInitializeSizes(SimStruct *S)
-        err = mdSystem->Step(x, xd, OutputAry, t, dt);
-
-        if (checkError(S)) return;
-        setOutputs(S, OutputAry);
+        //err = mdSystem->Step(x, xd, OutputAry, t, dt);
+        //if (checkError(S)) return;
+        //setOutputs(S, OutputAry);
```
Is there a reason this was commented out? I added this code in because it was what was causing MoorDyn to return 0 at t=0, which was breaking OpenFresco at the time.

aschellenberg74 committed:
apply bug fixes and cleanup based on feedback from Michael

aschellenberg74 commented:
Hey Michael, thanks for the feedback. I hope you had a good start at the new job.
1) I removed the dlls and libs.
2) I cleaned up the example folder.
3) I had forgotten to undo the commented-out initial conditions after I had finished testing different things. Thanks for catching that!

michaelcdevin commented:
Fantastic, this looks great! I'll approve and merge. Thank you for all your contributions to the project! :)
